components/Nav.tsx

```diff
@@ -134,193 +134,193 @@ const Nav: React.FC = () => {
     <>
       <a
         href="#content"
-        className="text-md absolute inset-x-0 -top-7 z-50 mx-auto w-40 -translate-y-12 transform rounded-lg border border-gray-300 bg-white px-3 py-2 text-center tracking-tight outline-none transition-transform duration-100 focus:translate-y-12 dark:border-neutral-700 dark:bg-black"
+        className="text-md absolute inset-x-0 -top-7 z-[120] mx-auto w-40 -translate-y-12 transform rounded-lg border border-gray-300 bg-white px-3 py-2 text-center font-semibold tracking-tight shadow-xl outline-none transition-transform duration-100 focus:translate-y-12 dark:border-neutral-700 dark:bg-black"
       >
         Skip to content
       </a>
+      <nav className="fixed left-0 top-0 right-0 z-[150] flex h-[74px] items-center border-b bg-white/80 backdrop-blur-[14px] transition dark:bg-primary/10">
```

Comment:
On dark theme, maybe could make the navbar more opaque compared to the light theme one

![image](https://user-images.githubusercontent.com/34704796/193830940-05bc9288-61a2-493f-bd0d-6d250c914124.png)

Light theme

![image](https://user-images.githubusercontent.com/34704796/193830971-becab7f4-1f63-496b-8b2c-645556ec31f9.png)

Reply:
One problem that occured when I did that is that the color looked off from the rest of the background...

I'll still try to make it more consistent.